OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -61,7 +61,7 @@ public void Run(ModData modData, string[] args)
 					var section = file.GetSection("TileSet{0:D4}".F(tilesetGroupIndex));
 
 					var sectionCount = int.Parse(section.GetValue("TilesInSet", "1"));
-					var sectionFilename = section.GetValue("FileName", "");
+					var sectionFilename = section.GetValue("FileName", "").ToLowerInvariant();
```

Comment:
I doubt it is a problem. As not defining `Images` leads to weird error messages I was mostly guessing. This doesn't break anything in practice.